LICENSE

```diff
@@ -296,15 +295,14 @@
    MIT License.
 
    Kyligence/kylinpy
-   ./clients/client-python/gravitino/utils/exceptions.py
    ./clients/client-python/gravitino/utils/http_client.py
 
    Apache Ranger
-   ./security-admin/src/main/java/org/apache/ranger/defines/JsonDateSerializer.java
-   ./security-admin/src/main/java/org/apache/ranger/defines/RangerCommonEnums.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VList.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VXDataObject.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VXGroup.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VXGroupList.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VXUser.java
-   ./security-admin/src/main/java/org/apache/ranger/view/VXUserList.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/JsonDateSerializer.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/RangerCommonEnums.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VList.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VXDataObject.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VXGroup.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VXGroupList.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VXUser.java
+   ./authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/reference/VXUserList.java
```

Comment:
Hi @liuchunhao thanks for your work. Can you please move this Ranger part under "Apache Arrow" and above "MIT License"? Thanks.

Reply:
Sure, I've adjusted it as requested.